kyo-prelude/shared/src/main/scala/kyo/Parse.scala

```diff
@@ -28,9 +37,9 @@ object Parse:
       * @return
       *   Parsed value if successful, drops the current parse branch if unsuccessful
       */
-    def read[A, S](f: Text => Maybe[(Text, A)] < S)(using Frame): A < (Parse & S) =
```

Comment:
I've removed the support for extra `S` effects in the read function since `Aspect` needs to know what effects will be used beforehand. It was also unused.